src/dbnode/storage/index/convert/convert.go

```diff
@@ -141,12 +144,35 @@ func FromSeriesIDAndTags(id ident.ID, tags ident.Tags) (doc.Document, error) {
 	return d, nil
 }
 
-// FromSeriesIDAndTagIter converts the provided series id+tags into a document.
-func FromSeriesIDAndTagIter(id ident.ID, tags ident.TagIterator) (doc.Document, error) {
+// FromRawSeriesIDAndTags converts the provided raw series id+encoded tags into a document.
```

Comment:
Can this add a note that it's safe to reuse the decoder since the document tag values will be cloned?

Reply:
👍